### ISSUE TITLE
Spike: Various a11y issues for MindTap

### ISSUE THREAD

**Issue by silvalaura** [cengage/react-magma#1748]
**Describe the bug**

> Hi! We received the Perkins results, what should be fixed in LRNO courses in the MindTap platform.
And after discussion with the team, it turns out that some issues should be fixed by the React Magma team
Should these tickets be moved to your project?
[NG-96812](https://jira.cengage.com/browse/NG-96812) Add missing alt attributes to images in Bulk flow **(significant)**
[NG-96759](https://jira.cengage.com/browse/NG-96759) Hide decorative icons from assistive technology in Bulk flow **(significant)**
[NG-96818](https://jira.cengage.com/browse/NG-96818) Announce Error messages in Bulk flow **(moderate)**
[NG-96823](https://jira.cengage.com/browse/NG-96823) Content should appear in meaningful sequence **(moderate)**
[NG-96824](https://jira.cengage.com/browse/NG-96824) Modal dialogs have a descriptive name in Bulk flow **(moderate)**

This team is using 4.4.0

**Acceptance Criteria**
Look through each of these issues and determine if:
1. Requires a fix on magma (look to see if we have any similar a11y bugs in the backlog already and mention them in the PR). Open a new ticket to be addressed in the future
2. Has been fixed on magma and required a MT version upgrade
3. Requires guidance for the MT team to fix on their end

Then, write this up and communicate with Yuliia ([thread](https://cengage.slack.com/archives/C8Y4FR63U/p1744811469891899))

The outcome of this ticket is a write up to document the solution for each of the issues above.

**New tickets:**

[NG-96812](https://jira.cengage.com/browse/NG-96812), [NG-96823](https://jira.cengage.com/browse/NG-96823) -> [A11y Fail: Alert - Add aria-label for icon](https://github.com/cengage/react-magma/issues/1766)

[NG-96759](https://jira.cengage.com/browse/NG-96759) -> [ A11y Fail: Radio Button - Hide decorative icons from assistive technology.](https://github.com/cengage/react-magma/issues/1765)

**Already fixed:**
[NG-96824](https://jira.cengage.com/browse/NG-96824) ->  was fixed within [A11y: Modal aria labels](https://github.com/cengage/react-magma/pull/1318) - need update to React Magma `4.5.0`  version or higher.

[NG-96818](https://jira.cengage.com/browse/NG-96818) -> was fixed within [Input: Error message not read by screenreader on Firefox when there is helper text](https://github.com/cengage/react-magma/issues/1661)  - need update to React Magma `4.5.0`  version or higher.

**Implementation issues:**

[NG-96823](https://jira.cengage.com/browse/NG-96823) -> the issue is the implementation into these pieces of code:
- https://stash.cengage.com/projects/MT/repos/mtevo-ui/pull-requests/1840/diff#client%2Fsrc%2Fmast%2Fcomponents%2FModals%2FBulkEditLatePolicyModal%2FBulkEditLatePolicyModal.jsx?t=128
- https://stash.cengage.com/projects/MT/repos/mtevo-ui/pull-requests/1840/diff#client%2Fsrc%2Fmast%2Fcomponents%2FModals%2FEmbeddedWarningMessage%2FEmbeddedWarningMessage.jsx

Need to update this implementation to fix the issue with the sequence of rendering some components.

https://perkins.caspio.com/dp/ca3640005e6f2f41f3634a75b225?ID=ID_0016670 -> the issue is with the implementation in this part of the code.

- https://stash.cengage.com/projects/MT/repos/mtevo-ui/browse/client/src/mast/components/Common/NodeInfo/M_NodeInfo.jsx#82

Need to update this implementation to fix the issue with grouping between the label and data.

**Comment by silvalaura** [comment]
A few others to investigate:

https://perkins.caspio.com/dp/ca3640005e6f2f41f3634a75b225?ID=ID_0016668
https://perkins.caspio.com/dp/ca3640005e6f2f41f3634a75b225?ID=ID_0016670

See slack thread for getting access into portal

**Comment by olehnoskov-cengage** [comment]
> A few others to investigate:
> 
> https://perkins.caspio.com/dp/ca3640005e6f2f41f3634a75b225?ID=ID_0016668 https://perkins.caspio.com/dp/ca3640005e6f2f41f3634a75b225?ID=ID_0016670
> 
> See slack thread for getting access into portal

Created a new ticket:
https://github.com/cengage/react-magma/issues/1769
